### PR TITLE
Update README - remove obsolete unreliable links for support

### PR DESCRIPTION
**Pull request by cevap**
Forum is bad place to send anybody to disucuss anything and should not be used. They ban and censor criticism which is in no way proper addressing of the issue.

staff closes instead to reply.

@sergiusens your forum is not proper place for disussion, you close each pull request because you do not want what? Let's talk here, as it is easiest, you can close this PR when we finish and John R Lenton should prove his words if you already claim that he is not a lier, I referenced to a post where he lied.

Beside that, how do you like choice of words by him "silencing". He seems to be really old in not knowing that I can register new account at any time and "silencing" is gone up to the point where you close your forum because you do not want new users, great goal istn it? 

You guys want to show the world how linux is easy and good in breaking it, claiming later falsely that it was broken where it wasnt and finally, silencing :) based on weather.

Snapcraft looked cool but all this in a pack just stinks, I cant trust you guys if you do break the code intentionally and it really should not be part of ubuntu then, especially if you do not care if pushing malicious code.

### CODE REVIEW

**Comment by cevap** [comment]
@jhenstridge I am not sure you get my private messages on your forum, please confirm with yes/no.

I saw your reply about manual verification. Thanks for that workaround but sadly it is useless for me. It is more than just one active snap which you checked, all are broken, maybe you do not see some because I disabled them for time being fixed.

Manual verification is in no way workaround for me, snapcraft is used to build on each commit from, launchpad builds 4 times a day etc... . 

For years when talking about distribution for specific projects snap was mainly no option because of many points but mainly security, considering that there is no "stable" snapcraft but that all are development versions and to be threaten as unstable, any sight in how you want to solve this problem?

You never replied in thread and your colleague banned then, where is the PR that was applied and why does it count for you as possibility to disable this check until you can ensure that there are no further bugs.

**Comment by cjp256** [comment]
@cevap Could you please provide some instructions to reproduce your specific issue?  We have some pending fixes in queue but I would like to make sure we are addressing the root of your problem.

**Comment by cevap** [comment]
@cjp256 hello, yes I can, what exactly would you require, git history has it all and snapcraft.yaml was written with simple scripts so that other projects can adapt it. Look in the history and snap builds when it started to be "wrong" version. Fix one project and other will be fixed too, but in first line, everything worked perfectly like it was before and now I see in thread that you allow again ${SNAP}/icon.png . 

You applied globally something that had neither clear error message nor your documentation was updated. If your team means that due to updates on snap you might apply again such untested things which means you do not care in first line about quality of service and actually using snap will only bring user in troubles, in crypto it is mostly directed to money and loss and that is why many struggle to use snap, but that should have been known to you together with other very well known issues.

Point now is actually to redefine what your service and real competences are, I am not satisfied and see no future for stable software deployment if untested code is pushed as stable and you have fully unqualified moderators there where you invite people to report to abuse them. You should consider enabling issues on github too.

As last, please explain word usage of your admin "silencing" as well as what it means, as he silence's as public person whose social accounts etc.. are all visible, as addition he makes bad attempt in provocation and finally he bans out of arguments after 3 offline posts. Can you please explain it in detail why you send there users and if we can expect that such incompetence will improve to something more "competent"? As last, those lies, insults and accusations etc.., can we look at them as official canonical statement? Any comment on your mods work?

**Comment by cjp256** [comment]
@cevap can you provide the output of your error?

Is this the commit that works around the issue for you?  Changing the paths of the .ico files?
https://github.com/ioncoincore/ion/commit/4f122c38cc07c4c0e8274c792a1fedd846de8040

**Comment by cevap** [comment]
@cjp256 I could but I am to lazy to look up now again due to your incompetent mod who deleted the thread and banned, could you please look up on forum, I hate to write same thing twice due to [idiots](https://github.com/chipaca), but as you try to help, this was when I wanted to see what happens if I use /usr/.. folder:
```
invalid icon path '${SNAP}/usr/share/icons/iond.ico'. Should either specify the basename of the file (with or without file extension), snap.<snap name>.<snap command>[.(png|svg)] or ${SNAP}/path/to/icon.(png|svg). If using snapcraft, consider using 'desktop: <file>' since the Icon paths in the desktop will be rewritten to use ${SNAP}/<file>. lint-snap-v2_desktop_file_icon (daemon.desktop, ${SNAP}/usr/share/icons/iond.ico) 
```

before that same error with ${SNAP}/iond.ico

Solution is to use png instead of icon. Somehow your team missed that too and you managed to delete whole thread reporting about it.



this should work but failed, it worked before properly and I always used ico and png due to some other reasons: https://github.com/ioncoincore/ion/commit/9309b80da92660b56db587f519330decbb9112b0

Can you please comment on your mods incompetent work and why do you enforce people to go there to be abused as well as like you seems officially state "silence"? I am very, very, very interesting in what kind of policies canonical now has, this does not sounds well :)

**Comment by cevap** [comment]
@chipaca and, how is your silencing succeeding? You must feel stupid now I guess, can you start replying now instead to let your colleagues get all the critics and trying to paint some other picture, you ban then you are silent, I ask myself now who has silenced who, what do you say?

**Comment by sergiusens** [comment]
@cevap please refrain from calling people incompetent or insulting them. This is the reason you were temporarily silenced on the forum. That is the place to discuss issues, not here..

I read the thread discussed on the forum with fresh eyes today only to discover that the person that is insulting others, remember that there are people behind this project, is **you** and yet on comments in totally unrelated PRs you bring up the issue of having been insulted? This makes no sense. This behavior is very unprofessional.

Mistakes happen all the time and the best way to reach consensus and a good resolution is by treating people with respect.

Bottom line is that for this project, the forum is still the best place to discuss issues, for which, I am closing this PR, please do not open a new one.

**Comment by cevap** [comment]
@sergiusens I already asked you not to teach me how to behave or reply to insults, live your life
